### ISSUE TITLE
Investigate Text-2-Speech and integrate with T2V

### ISSUE THREAD

**Issue by yuanxion** [yuanxion/Text2Video-Zero#19]
As descripted [here](https://github.com/yuanxion/Text2Video-Zero/issues/18#issuecomment-1656700724), we might want to integrate a TTS with T2V to get a more attractive application to users.

There is a TTS repo https://github.com/coqui-ai/TTS, which might be a good candidate to start with.
Other TTS repo: https://github.com/mozilla/TTS.

**Comment by yuanxion** [comment]
- OpenAI Whisper + FFmpeg + TTS：动态实现跨语言视频音频翻译
https://zhuanlan.zhihu.com/p/631644803

- edge-tts [Good]

```
output_file="mytest.mp3"
edge-tts --list-voices | grep zh
  Name: zh-CN-XiaoxiaoNeural
  Name: zh-CN-XiaoyiNeural
  ...
edge-tts --voice zh-CN-XiaoyiNeural \
--text "春天來了。池塘里的冰融化了。青蛙媽媽睡了一個冬天。" \
--write-media $output_file --write-subtitles $output_file.vtt
```


- 一个文本转视频的工具
https://github.com/bravekingzhang/text2video
https://github.com/bravekingzhang/text2video/blob/main/text_to_video.py
<img width="715" alt="image" src="https://github.com/yuanxion/Text2Video-Zero/assets/96522341/b320a5f0-afb7-46a9-9e94-d3e710ee59c9">

**Comment by yuanxion** [comment]
Addressed with PR #21.
